source/fe/mapping_q.cc

```diff
@@ -1085,6 +1085,26 @@ void MappingQ<dim,spacedim>::transform
   MappingQ1<dim,spacedim>::transform(input, output, *q1_data, mapping_type);
 }
 
+template<int dim, int spacedim>
+void MappingQ<dim,spacedim>::transform
+(const VectorSlice<const std::vector< DerivativeForm<2, dim, spacedim> > > input,
+ VectorSlice<std::vector<Tensor<3,spacedim> > >             output,
+ const typename Mapping<dim,spacedim>::InternalDataBase &internal,
+ const MappingType type) const
+{
+  this->MappingQ1<dim,spacedim>::transform(input, output, internal, type);
```

Comment:
@maieneuro Thanks for the confirmation.

@tjhei I'm resorting arguments to the transform() functions as we speak. This would yield a merge conflict. Please just drop this one commit and I'll add the correct function declarations to on my branch.